src/index.js

```diff
@@ -106,9 +111,44 @@ function displayStartButton() {
 }
 
 guessButton.on('click', () => {
+  if (game.roundCount < 3) {
   regularTurn.updateScore(guessInput.val());
   guessInput.val('');
-  if (regularRound.answerStrings.every(answer => answer === 'false')) {
+    if (regularRound.answerStrings.every(answer => answer === 'false')) {
     instantiateRoundAndTurn();
+    }
+  }
+  if (game.roundCount === 3) {
+    dominationRound = new DominationRound(game);
+    dominationRound.beginDominationTurn();
+    dominationTurn = new DominationTurn(dominationRound);
+    dominationTurn.updateScore(guessInput.val());
+    guessInput.val('');
+  }
+});
+
+multiplier.on('keydown', () => {
+  timer.show();
+  goButton.show();
+})
+
+goButton.on('click', () => {
+  timeLeft = 30;
+  startTimer();
+})
+
+const startTimer = () => {
+  const counter = setInterval(countdown, 1000);
+
+  function countdown() {
+    timeLeft -= 1;
+
+    if (timeLeft < 0) {
+       clearInterval(counter);
+       dominationRound.endDominationRound();
+       return;
```

Comment:
Interesting method, I like it.
I was thinking something very similar in my head.